eng/Version.Details.xml

```diff
@@ -383,11 +383,6 @@
       <Uri>https://github.com/dotnet/roslyn-analyzers</Uri>
       <Sha>b4d9a1334d5189172977ba8fddd00bda70161e4a</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.ApiCompat.Task" Version="8.0.100-rtm.23520.8">
```

Comment:
Why did you decide to remove this?

Reply:
We don't need live toolset updates from the SDK anymore. We stick with the 8.0.100 version.